README.md

```diff
@@ -1,29 +1,92 @@
 
-## Learning Objectives (Students will be able to ....) 
-- Describe a tuple as being a type that groups multiple values into a single compound value.
-- Explain that values within a tuple can be of any type and do not have to be of the same type as each other.
-- Iterate over the key-value pairs in a dictionary where each item is returned as a (key, value) tuple.
-- Create a tuple and name the individual elements
 
-## What the Student Already Knows
-A student should already understand the concept of variables, constants, Int, String, Bool, Array, Dictionary and functions but not Tuples.
+A Game of Chance
+----------------
+ Suppose you were creating a game that included the use of dice. You will randomly roll a pair a dice to receive some prize or to advance in the game. With our current knowledge of functions, the actions to start the game may look something like this:
 
-## Outline/ Notes
 
-* Start with the known- students know how to iterate over arrays. Show an example of iterating over an array. 
-    * Present some real world problem such as, “If you have a dictionary where the keys represent the seasons and the values represent your favorite color associated with that season, is there an easy way to print out a list that for each sentence and color that says “In the {season} my favorite color is {color}.”
-    - If you have a better example feel free to use it, but always aim for a real world example. And you can use that example throughout the rest of your reading and/or use other new examples. 
-    * The answer is tuples and this sets the stage for writing the content that covers your learning objectives above.--- You can iterate over the key-value pairs in a dictionary with a for-in loop. Each item in the dictionary is returned as a (key, value) tuple. Explain how you can decompose the tuple’s members into temporary constants as part of the iteration.
+```swift
+
+func startGame() {
+    let dice: Int = throwDice()
+    print("Congrats - you rolled \(dice)!")
+}
+
+//simulate  dice throw
+func throwDice() -> Int {
+    print("..calculating random results from die throw..")
     
+    //generate and return a random number - let's just return 5 for now
+    return 5 
+}
+
+startGame()
+
 ```
+
+
+Even though we accurately applied what we know, we’ve encountered some issues.  Our requirements state the game involves the throwing of `two` dice, but our `throwDice` function only allows for `one` dice to be thrown at a time. While we could call the function twice to generate `two` random numbers, it would be ideal if we could somehow call `throwDice` once and have it  return `two` independent values.
+ 
+
+Upon further examination, the `throwDice` return value of `Int` also could cause issues related to readibility. For example, a future developer may wonder if the `Int` return value is indeed the result of the simulated dice throw, or the result of some other calculation.
+
+
+Tuples as Types
+---------------
```

Comment:
I like the way you grouped the various things that make up a tuple. It makes it clear to the student what it is they are learning and serves as a good reference if they ever have to come back to the readme to brush up on what tuples are.